indico/modules/events/papers/controllers/management.py

```diff
@@ -226,3 +227,26 @@ def _process(self):
                                num_not_submitted_papers).format(num=num_not_submitted_papers), 'warning')
             return jsonify_data(**self.list_generator.render_list())
         return jsonify_form(form=form, submit=_('Judge'), disabled_until_change=False)
+
+
+class RHManageReviewingSettings(RHManagePapersBase):
+    def _process(self):
+        defaults = FormDefaults(content_review_questions=self.event_new.cfp.content_review_questions,
+                                layout_review_questions=self.event_new.cfp.layout_review_questions,
+                                **paper_reviewing_settings.get_all(self.event_new))
+        form = PaperReviewingSettingsForm(event=self.event_new, obj=defaults)
+        if form.validate_on_submit():
+            data = form.data
+            content_review_questions = data.pop('content_review_questions', [])
+            layout_review_questions = data.pop('layout_review_questions', [])
+            self.event_new.paper_review_questions = content_review_questions + layout_review_questions
+
+            for key, value in data.pop('email_settings').iteritems():
+                data[key] = RoleConverter.to_python(value) if isinstance(value, list) else value
```

Comment:
cause the value can be a list which needs to be converted to a set. Instead of reimplementing the logic of the `RoleConverter`, I just used it.